### ISSUE TITLE
Add metrics to extensions and other DB objects (instructions, CSRs, etc.) of their completion status

### ISSUE THREAD

**Issue by james-ball-qualcomm** [riscv-software-src/riscv-unified-db#118]
Derek, can you come up with the metrics for the various DB objects? Then I can add it and include it in backends as appropriate. Or even have a separate backend just to generate the information.

**Comment by dhower-qc** [comment]
Yep. As we discussed, this will be more straightforward once we have the bare-bones of all instructions (so we know that 100% is). @AFOliveira is working on that.

**Comment by dhower-qc** [comment]
Added a dashboard in #207

**Comment by ThinkOpenly** [comment]
Can this be closed?
